src/dialects/sqlite3/index.js

```diff
@@ -57,6 +57,10 @@ assign(Client_SQLite3.prototype, {
     return new SQLite3_DDL(this, compiler, pragma, connection)
   },
 
+  wrapIdentifier: function wrapIdentifier(value) {
```

Comment:
the way mysql does it nowadays is:

```js
  wrapIdentifier(value) {
    return (value !== '*' ? `\`${value.replace(/`/g, '``')}\`` : '*')
  },
```
did you copy-paste compiled code? Copy-pasting just fine in this case though :)